raiden/exceptions.py

```diff
@@ -170,7 +170,7 @@ class RaidenServicePortInUseError(RaidenError):
     """Raised when Raiden service port is already in use"""
 
 
-class InvalidDBData(RaidenError):
+class InvalidDBData(RaidenUnrecoverableError):
```

Comment:
@LefterisJP with the mainnet switch, please let this exception crash the node.

Reply:
Done: https://github.com/raiden-network/raiden/pull/2530/commits/016ff70d57398c4d33f5fd4514734157d9a17464